tests/test_progress.py

```diff
@@ -3,33 +3,32 @@
 import io
 import os
 import tempfile
-from time import sleep
 from types import SimpleNamespace
 
 import pytest
 
 import rich.progress
-from rich.progress_bar import ProgressBar
```

Comment:
only changed these imports in order to pass the `isort` part of the pre-commit hook :slightly_smiling_face: